chacra/controllers/health.py

```diff
@@ -0,0 +1,12 @@
+from pecan import expose, abort
+
+from chacra.async import checks
+
+
+class HealthController(object):
+
+    @expose('json')
+    def index(self):
+        if checks.is_healthy():
+            return "Everything OK"
```

Comment:
I don't think we need to return anything here. Just the abort below *if it is not healthy*.

The controller will respond with a 2XX response if there is no body